self_enrollment_tool/templates/self_enrollment_tool/base.html

```diff
@@ -16,6 +16,12 @@
 
 {% block javascript %}
     {{ block.super }}
+    <!-- jQuery and Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    
```

Comment:
The [icommons_ui/base_lti.html](https://github.com/Harvard-University-iCommons/django-icommons-ui/blob/master/icommons_ui/templates/icommons_ui/base_lti.html) template already has the following:
```
<script src="https://static.tlt.harvard.edu/shared/jquery-3.6.3/jquery-3.6.3.min.js"></script>
<script src="https://static.tlt.harvard.edu/shared/bootstrap-5.3.0/js/bootstrap.bundle.min.js"></script>
```
Do you need these versions for the date popper to work?
Do we already have a version of the date popper in our static S3 bucket? If not we should add it there and reference it as part of the static.tlt.harvard.edu/shared/ endpoint

Reply:
I added these to get the Jquery copy link and delete icons to work properly. The date popper was working properly before these were added.